packages/transcript-view/stories/index.stories.js

```diff
@@ -0,0 +1,44 @@
+import {
+  storiesOf,
+  html,
+  withKnobs,
+  color,
+  text,
+  boolean,
+  number,
+  withClassPropertiesKnobs,
+} from '@open-wc/demoing-storybook';
+
+import TranscriptView from '../lib/src/transcript-view';
+import TranscriptEntryConfig from '../lib/src/models/transcript-entry-config';
+import transcript from './transcript';
+
+const convertedTranscript = transcript.map((entry) => {
+  return new TranscriptEntryConfig(
+    entry.id, entry.start, entry.end, entry.text, entry.is_music || false, entry.search_match_index);
+})
+
+console.log(convertedTranscript);
```

Comment:
**Suggestion**: remove logs or replace with a system like Mitra implemented for debugging in the dweb components.